src/containers/channelDetail/ChannelDetailContainer.jsx

```diff
@@ -14,13 +14,17 @@ import { ALL_STATUSES } from "../../constants/uiConstants";
 
 class ChannelDetailContainer extends Component {
   async componentDidMount() {
-    if (this.props.selectedSuggestion) {
+    const selectedSuggestion = this.getSelectedSuggestion();
+    if (selectedSuggestion) {
+      if (this.props.selectedSuggestion) {
+        sessionStorage.setItem("tempSelectedSuggestion", JSON.stringify(this.props.selectedSuggestion));
+      }
       const actualDate = new Date();
       let oneMonthBefore = new Date().setMonth(actualDate.getMonth() - 1);
       const oneMonthBeforeDate = new Date(oneMonthBefore);
```

Comment:
I think this variables are unused now. If I'm wrong, I'd changed `actualDate` to`currentDay`

Reply:
they are used to contruct `oneMonthBeforeDate` value used on `filterFromDate: oneMonthBeforeDate,`